code/src/components/ThoughtsList.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react"
+
+import { useEffect } from "react"
+import { ThoughtCard } from "./ThoughtCard"
+
+export const ThoughtsList = () => { 
+    const ThoughtsUrl= "https://happy-thoughts-technigo.herokuapp.com/thoughts";
+    const [thoughts, setThoughts] = useState([]);
+
+    // useEffect prevents the infinite looping of creating new arrays after the data is fetched. 
+    useEffect (() => { 
+      fetchThoughts()
+    }, []);
+
+    const fetchThoughts = () => {
```

Comment:
Unfortunately I missed wednesday's lecture where the fetches were made into a component that would pass onto info to others and because my time was limited this week I was too far on the project to change it and since Van said on friday it was okay I left it like this but yes it does make sense 😊